builder/config.go

```diff
@@ -15,16 +15,47 @@ import (
 type Config struct {
 	Description string            `toml:"description"`
 	Buildpacks  []BuildpackConfig `toml:"buildpacks"`
-	Groups      []GroupMetadata   `toml:"groups"`
+	Order       OrderConfig       `toml:"order"`
 	Stack       StackConfig       `toml:"stack"`
 	Lifecycle   LifecycleConfig   `toml:"lifecycle"`
 }
 
+type OrderConfig []GroupConfig
+
+func (o OrderConfig) ToMetadata() OrderMetadata {
+	var order OrderMetadata
+	for _, gp := range o {
+		var buildpacks []BuildpackRefMetadata
+		for _, bp := range gp.Group {
+			buildpacks = append(buildpacks, BuildpackRefMetadata{
+				ID:       bp.ID,
+				Version:  bp.Version,
+				Optional: bp.Optional,
+			})
+		}
+
+		order = append(order, GroupMetadata{
+			Buildpacks: buildpacks,
+		})
+	}
+
+	return order
+}
+
+type GroupConfig struct {
+	Group []BuildpackRefConfig `toml:"group"`
+}
+
+type BuildpackRefConfig struct {
```

Comment:
This is exactly the same as `BuildpackMetadata` and `BuildpackRefMetadata`. I think we just need one `BuldpackRef`; that would be less confusing.

Reply:
It could also be `GroupEntry` or something like that